stable/database/files/nuosm

```diff
@@ -415,6 +478,12 @@ if [ "$SEPARATE_JOURNAL" == "true" ]; then
   log "journal-dir=${JOURNAL_DIR}"
 fi
 
+if [ "$SNAPSHOT_RESTORED" == "true" ]; then
+  loadFromSnapshot
+else
+  log "No snapshot"
```

Comment:
Is this for diagnostics? I think we should log something if we are relocating the snapshot archive/journal and be silent otherwise.

Reply:
I like this as a debug log. I will remove it.